internal/pkg/plugin/trello/read.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func Read(options map[string]interface{}) (map[string]interface{}, error) {
+
```

Comment:
There is an unnecessary blank line here; but it doesn't matter, remember to modify it in the next pr